app/helpers/application_helper/form_tags.rb

```diff
@@ -1,5 +1,15 @@
 module ApplicationHelper
   module FormTags
+    # override: default password_field_tag doesn't switch off autocomplete
+    def password_field_tag(name, value = nil, options = {})
+      options = {
+        :type         => 'password',
+        :autocomplete => 'off',
+      }.merge(options)
```

Comment:
```
options.merge!
```

Reply:
not if you want it to be possible to override this